Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -62,6 +65,7 @@ @interface ConversationViewItem ()
 @property (nonatomic, readonly, nullable) NSString *quotedRecipientId;
 @property (nonatomic, nullable) TSAttachmentStream *attachmentStream;
 @property (nonatomic, nullable) TSAttachmentPointer *attachmentPointer;
+@property (nonatomic, nullable) OWSContact *contactShare;
```

Comment:
This is fine for now, but I'm guessing we'll eventually replace this DB record with a view model similar to the way we handled quoted replies, so we can fetch any contact attachment meta-data (TSAttachmentStream) as part of ConversationViewItem hydration.

Reply:
Yes, I've been wondering if that's going to be necessary.